### PR TITLE
Rich activity log descriptions

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/5881
Require https://github.com/opencollective/opencollective-api/pull/7842 

image

### CODE REVIEW

**Comment by Betree** [comment]
@SudharakaP I'm thinking about merging this first patch then iterating on other missing transactions & adding more links later. It still needs to be rebased to solve the lining issue after we merge https://github.com/opencollective/opencollective-api/pull/7880 (also ready for review)

File: lib/i18n/activities.js [hunk 1]
```diff
@@ -12,37 +12,75 @@ export const ActivityLabelI18n = defineMessages({
...
+    defaultMessage: 'Paid <Expense>expense</Expense> on <Account></Account>',
+  },
+  COLLECTIVE_EXPENSE_MARKED_AS_UNPAID: {
+    defaultMessage: 'Marked <Expense>expense</Expense> as unpaid',
+  },
+  COLLECTIVE_EXPENSE_APPROVED: {
+    defaultMessage: 'Approved <Expense>expense</Expense> for <Account></Account>',
+  },
+  COLLECTIVE_EXPENSE_UNAPPROVED: {
+    defaultMessage: 'Approved <Expense>expense</Expense> for <Account></Account>',
```
Comment (SudharakaP) [inline lib/i18n/activities.js:51]:
```suggestion
    defaultMessage: 'Unapproved <Expense>expense</Expense> for <Account></Account>',
```

**Review by SudharakaP — APPROVED** [review]
Works well, with some minor feedback. 👍🏽

File: lib/i18n/activities.js (hunk 1 above)
Comment (Betree) [reply lib/i18n/activities.js:51]: Good catch :100:

**Pull request merged**